Add 'headers' option to docs (#18814)
dotnet/AspNetCore.Docs: aspnetcore/signalr/configuration.md

| JavaScript Option | Default Value | Description |
| ----------------- | ------------- | ----------- |
| `accessTokenFactory` | `null` | A function returning a string that is provided as a Bearer authentication token in HTTP requests. |

> | `headers` | `null` | Dictionary of headers to be sent with every HTTP request. Sending headers in the browser will not work for WebSockets or the ServerSentEvents stream. |

Reviewer: ```suggestion
| `headers` | `null` | Dictionary of headers sent with every HTTP request. Sending headers in the browser doesn't work for WebSockets or the <xref:Microsoft.AspNetCore.Http.Connections.HttpTransportType.ServerSentEvents> stream. |
```

> | `logMessageContent` | `null` | Set this to `true` to log the bytes/chars of messages being sent and received by the client. |

Reviewer: ```suggestion
| `logMessageContent` | `null` | Set to `true` to log the bytes/chars of messages sent and received by the client. |
```

| JavaScript Option | Default Value | Description |
| ----------------- | ------------- | ----------- |
| `accessTokenFactory` | `null` | A function returning a string that is provided as a Bearer authentication token in HTTP requests. |

> | `logMessageContent` | `null` | Set this to `true` to log the bytes/chars of messages being sent and received by the client. |

Reviewer: ```suggestion
| `logMessageContent` | `null` | Set to `true` to log the bytes/chars of messages sent and received by the client. |
```

| JavaScript Option | Default Value | Description |
| ----------------- | ------------- | ----------- |
| `accessTokenFactory` | `null` | A function returning a string that is provided as a Bearer authentication token in HTTP requests. |

> | `logMessageContent` | `null` | Set this to `true` to log the bytes/chars of messages being sent and received by the client. |

Reviewer: ```suggestion
| `logMessageContent` | `null` | Set to `true` to log the bytes/chars of messages sent and received by the client. |
```

| JavaScript Option | Default Value | Description |
| ----------------- | ------------- | ----------- |
| `accessTokenFactory` | `null` | A function returning a string that is provided as a Bearer authentication token in HTTP requests. |

> | `logMessageContent` | `null` | Set this to `true` to log the bytes/chars of messages being sent and received by the client. |

Reviewer: ```suggestion
| `logMessageContent` | `null` | Set to `true` to log the bytes/chars of messages sent and received by the client. |
```

> | `logMessageContent` | `null` | Set this to `true` to log the bytes/chars of messages being sent and received by the client. |

Reviewer 2: Nice catch seeing `logMessageContent` was undocumented. It looks like it's been there since 1.0, so it could be added in the "< aspnetcore-2.2" moniker range.

https://github.com/aspnet/SignalR/blob/e1a22b315dae5fc1335114908a545cd706af6c60/clients/ts/signalr/src/IHttpConnectionOptions.ts#L30-L34

Author: Oh whoops, thought I added it to them all
